### PR TITLE
added Britishness filter

### PR DESCRIPTION
**Pull request by james-simon**
This filter identifies passages using words unique to British English.

### CODE REVIEW

File: filters/britishness/filter.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+
+class BritishnessFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self, britwords_required=1):
+        super().__init__()
+        self.britwords_required = britwords_required
+        # self.keywords = keywords
```
Comment (aadesh11) [inline filters/britishness/filter.py:17]: Please remove commented code.

File: filters/britishness/filter.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
+import json
+import re
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from initialize import spacy_nlp
+import spacy
+
+
+class BritishnessFilter(SentenceOperation):
```
Comment (aadesh11) [inline filters/britishness/filter.py:10]: Please add Docstrings for your filter class.

File: filters/britishness/filter.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+def get_britwords():
+    britwords = []
+
+    spelling_json_file = open('filters/britishness/spelling_map.json')
+    spelling_map = json.load(spelling_json_file)
+
+    britwords += spelling_map.values()
+
+    vocab_map = {'Scotch tape': 'sellotape',
```
Comment (aadesh11) [inline filters/britishness/filter.py:38]: For cleaner code, better to move this map also in a JSON file, something like below:

{
  "vocab": {
    'Scotch tape': 'sellotape'
  },
  "spelling": {
    "accessorize": "accessorise"
  }
}

File: filters/britishness/filter.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+     'upset': 'miffed',
+     'vacation': 'holiday',
+     'vacuum cleaner': 'hoover',
+     'windshield': 'windscreen',
+     'wrench': 'spanner'
+    }
+
+    britwords += vocab_map.values()
+
+    britslang = ['tosser', 'blimey', 'wanker', 'chuffed', 'sod off', 'wonky', 'whinge', 'tenner', 'fiver', 'quid', 'toff', 'skive', 'scouse', 'scouser', 'cockney', 'nicked', 'nutter', 'gobsmacked', 'chap', 'bugger', 'anticlockwise', 'anti-clockwise', 'nosh', 'bollocks', 'ponce', 'bangers', 'telly', 'knickers', 'uni', 'albion', 'chunder', 'fortnight', 'grockel', 'kerfuffle', 'scrummy']
```
Comment (aadesh11) [inline filters/britishness/filter.py:146]: Please split the list into multiple lines instead of having a big single line code.

**Comment by richplant** [comment]
Can I suggest you rename this to 'Englishness filter', since it doesn't exactly conform to Scottish, Welsh, or Northern Irish English dialects.

**Comment by james-simon** [comment]
Thanks for the feedback, @aadesh11 and @richplant! I believe I've made all your suggested changes.

**Review by aadesh11 — APPROVED** [review]

**Comment by kaustubhdhole** [comment]
@richplant anymore comments or is this good in the current state ?

**Comment by richplant** [comment]
No further comments, all good.

**Pull request merged**